web/src/App.res

```diff
@@ -245,43 +306,61 @@ module App = {
     let (showAbout, setShowAbout) = React.useState(_ => false)
     let (showLoginModal, setShowLoginModal) = React.useState(_ => false)
 
+    // Remove the token 60s before its expiration
```

Comment:
Would it be possible to start the setTimeout right when the token is read from the local storage? I wary of the `useEffect0` semantic...

Reply:
I tried but within the store's create function I'm not able to initialize the setTimeout cb with the dispatch action.

So if you know how to do this, please let me know if not let's move forward with this change.